service/matching/physical_task_queue_manager_test.go

```diff
@@ -308,7 +312,6 @@ func TestLegacyDescribeTaskQueue(t *testing.T) {
 	require.NotNil(t, taskQueueStatus)
 	require.Zero(t, taskQueueStatus.GetAckLevel())
 	require.Equal(t, taskCount, taskQueueStatus.GetReadLevel())
-	require.Equal(t, taskCount, taskQueueStatus.GetBacklogCountHint())
```

Comment:
shouldn't we still keep this assertion to verify that a non-zero hint is returned? (at the end you only verify the 0 case)

Reply:
Agreed. Added.